src/runtime_src/tools/scripts/service_bundle.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2019-2021 Xilinx, Inc. All rights reserved.
+#
+# This script collects debugging logs for XRT managed sub-systems
+
+TIME=`date +%F-%T`
+ID=`id -un`
+FULL_CMD=${BASH_SOURCE[0]}
+BASE_CMD=`basename "$0"`
+
+progress() {
+	echo -n "running: $func - progress: $(($1+1))/$2"
+	#echo -ne "running: $func - progress: \b\b\b$(($1+1))/$(($2+1))\b\b\b"
```

Comment:
There shouldn't be any dead code.  

If this functionality is needed, then an helper function should be created and depending on an option or environment variable then the alternate message should be produced.

Reply:
removed